exporter/sematextexporter/README.md

```diff
@@ -0,0 +1,54 @@
+# Sematext Exporter
+<!-- status autogenerated section -->
+<!-- end autogenerated section -->
+
+This exporter supports sending metrics to [Sematext Cloud](https://sematext.com/) in Influx line protocol format
+
+## Configuration
+
+The following configuration options are supported:
+
+* `endpoint` (required) HTTP/S destination for metric receivers. It is dependent on the region:
+    - US: `spm-receiver.sematext.com`
+    - EU: `spm-receiver.eu.sematext.com`
+* `timeout` (default = 5s) Timeout for requests
+* `AppToken` App token is the token of Sematext Monitoring App to which you want to send the metrics.
+* `Region`  Region specifies the Sematext region the user is operating in; must be one of:
+  * `US`
+  * `EU`
+* `payload_max_lines` (default = 10_000) Maximum number of lines allowed per HTTP POST request
+* `payload_max_bytes` (default = 10_000_000) Maximum number of bytes allowed per HTTP POST request
+* `metrics_schema` (default = telegraf-prometheus-v2) The chosen metrics schema to write; must be one of:
+    * `otel-v1`
+    * `telegraf-prometheus-v2`
```

Comment:
@Eromosele-SM probably only one schema in the end. Also, this Exporter will end up shipping telemetry other than metrics. So maybe you want to structure the README in preparation for that. For example, this particular schema will be only for metrics.

Reply:
I can always edit the readme when we have logs support ready?

Reply:
Ofc. I hope the YAML is also structured in anticipation of Exporter sending other telemetry later (so other App tokens, other endpoints, although the region could probably remain general).